moha/hamiltonians.py

```diff
@@ -677,3 +707,167 @@ def __init__(self,
             J_ax=J_ax,
             connectivity=connectivity
         )
+
+
+class HamTJUV(HamPPP, HamHeisenberg):
+    r"""t-J-U-V Hamiltonian."""
+
+    def __init__(self,
+                 connectivity_ppp: Union[list, np.ndarray],
+                 connectivity_heisenberg: np.ndarray,
```

Comment:
I think there is only one type of connectivity that needs to be provided. If connectivity is given as list of tuples, adjacency matrix needs to be build first. Once build, it can be used to populate "Heisenberg" part